stl/inc/xthreads.h

```diff
@@ -126,6 +125,7 @@ int __cdecl _Smtx_try_lock_exclusive(_Smtx_t*);
 int __cdecl _Smtx_try_lock_shared(_Smtx_t*);
 void __cdecl _Smtx_unlock_exclusive(_Smtx_t*);
 void __cdecl _Smtx_unlock_shared(_Smtx_t*);
+void __cdecl _Thrd_sleep_for(long long /*ns*/); // TRANSITION: defined in sharedmutex.cpp for convenience

```

Comment:
I'm questioning having `TRANSITION` in this comment.
I really don't think that a single function deserves its own `.cpp` file.
Later on with vNext this will be moved to the usual static/dynamic STL Library rather than Import Library, but this is implied for entire or nearly entire Import Library, so no extra `TRANSITION` comment needed.

Reply:
I've removed `TRANSITION`.